src/xcode/ENA/ENA/Source/Workers/WebPageHelper.swift

```diff
@@ -37,7 +37,17 @@ enum WebPageHelper {
 		}
 	}
 
-	static func openSafari(withUrl url: URL, from viewController: UIViewController) {
+	static func open(withUrl url: URL, from viewController: UIViewController) {
+		switch url.scheme {
+		case "tel", "mailto":
+			UIApplication.shared.open(url, options: [:], completionHandler: nil)
+		default:
+			openLink(withUrl: url, from: viewController)
+		}
+
+	}
```

Comment:
Semantically this is a little weird. It is a `WebPageHelper`. However it also handles phone and mail links. Would be cleaner to extract into a separate helper.

Reply:
Totally agree, this should be moved outside, either that or the helper should be renamed.

Reply:
I renamed the helper 👍